agir/front/components/genericComponents/ModalConfirmation.js

```diff
@@ -50,6 +50,7 @@ const ModalConfirmation = (props) => {
     dismissLabel,
     confirmationLabel = "",
     confirmationUrl = "",
+    confirmationFunc = null,
```

Comment:
```suggestion
    onConfirm,
```
Pour garder le format `on**` qu'on utilise normalement pour les props qui sont des fonctions